### ISSUE TITLE
Option similar to ruby_indent_assignment_style

### ISSUE THREAD

**Issue by cvincent** [rhysd/vim-crystal#42]
The Ruby Vim plugin includes an option called `ruby_indent_assignment_style` which determines the behavior of indenting after an assignment, commonly with `begin`, `if`, etc. Two different styles are shown below, each of which is supported by this option:

```ruby
@var = if some_condition
  true
else
  false
end

###

@var = if some_condition
         true
       else
         false
       end
```

I greatly prefer the first style, but `vim-crystal` only appears to support the latter style. I'd love to see this option added.

**Comment by rhysd** [comment]
I agree that the feature looks useful. But it should be provided by formatter because vim-crystal uses official formatter to format source code. Could you request it in [crystal repo](https://github.com/crystal-lang/crystal)? When the feature is added to the formatter, vim-crystal would automatically gain the feature.

**Comment by cvincent** [comment]
Done: https://github.com/crystal-lang/crystal/issues/5134
